internal/controller/drclusterconfig_controller.go

```diff
@@ -272,12 +288,71 @@ func (r *DRClusterConfigReconciler) processCreateOrUpdate(
 		return ctrl.Result{Requeue: true}, err
 	}
 
+	if err := r.validateS3Profiles(ctx, log, drCConfig); err != nil {
+		log.Info("Reconcile error", "error", err)
+		setDRClusterConfigConfigurationProcessedCondition(&drCConfig.Status.Conditions, drCConfig.Generation,
+			err.Error(), metav1.ConditionFalse, DRClusterConfigS3Unreachable)
+
+		return ctrl.Result{Requeue: true}, err
+	}
+
 	setDRClusterConfigConfigurationProcessedCondition(&drCConfig.Status.Conditions, drCConfig.Generation,
 		"Configuration processed and validated", metav1.ConditionTrue, DRClusterConfigConditionConfigurationProcessed)
 
 	return ctrl.Result{}, nil
 }
 
+func (r *DRClusterConfigReconciler) validateS3Profiles(ctx context.Context, log logr.Logger,
+	drCConfig *ramen.DRClusterConfig,
+) error {
+	// Fetch the ramen config resource
+	_, ramenConfig, err := ConfigMapGet(ctx, r.Client)
+	if err != nil {
+		return fmt.Errorf("config map get: %w", err)
```

Comment:
Update error message like so (just going by other cases in the code):
`failed to get Ramen configmap` instead of `config map get`